spur/ssh.py

```diff
@@ -80,7 +80,14 @@ def generate_run_command(self, command_args, store_pid,
             commands.append("echo $$")
 
         if cwd is not None:
-            commands.append("cd {0}".format(escape_sh(cwd)))
+            # 1. Change directory before to check if the command exists
+            # (using command || which) because it might be in this
+            # directory:
```

Comment:
I just wanted to emphasize that we have to change directory before to check if the command exists.